pkg/container/module.go

```diff
@@ -308,3 +308,102 @@ func UninstallContainerd(m *UninstallContainerModule) []task.Interface {
 		disableContainerd,
 	}
 }
+
+type CriMigrateModule struct {
+	common.KubeModule
+
+	Skip bool
+}
+
+func (i *CriMigrateModule) IsSkip() bool {
+	return i.Skip
+}
+
+func (p *CriMigrateModule) Init() {
+	p.Name = "CriMigrateModule"
+	p.Desc = "Cri Migrate manager"
+
+	if p.KubeConf.Arg.Type == common.Docker && p.KubeConf.Arg.Role == common.Worker {
+		p.Tasks = MigrateToDocker(p)
+	} else if p.KubeConf.Arg.Type == common.Docker && p.KubeConf.Arg.Role == common.Master {
+		p.Tasks = MigrateMToDocker(p)
+	} else if p.KubeConf.Arg.Type == common.Conatinerd && p.KubeConf.Arg.Role == common.Worker {
+		p.Tasks = MigrateToContainerd(p)
+	} else if p.KubeConf.Arg.Type == common.Conatinerd && p.KubeConf.Arg.Role == common.Master {
+		p.Tasks = MigrateMToContainerd(p)
```

Comment:
Recommended use：
```go
if p.KubeConf.Arg.Type == common.Docker {
        if p.KubeConf.Arg.Role == common.Worker {
		// xxx
	}
        if p.KubeConf.Arg.Role == common.Master {
		// xxx
	}
}
if p.KubeConf.Arg.Type == common.Conatinerd {
        if p.KubeConf.Arg.Role == common.Worker {
		// xxx
	}
        if p.KubeConf.Arg.Role == common.Master {
		// xxx
	}
}

```

Reply:
Yes, it is better, I will optimize it, thank you